web/templates/play/show.html.eex

```diff
@@ -1,2 +1,4 @@
 <%= render "_modal_keybinds.html" %>
 <div id="board-viewer" class="play-show"></div>
+
+<%= link "Explodey View", to: skin_path(@conn, :show, @game), class: "btn btn-default btn-xs view-btn" %>
```

Comment:
It might necessary to switch if you accidentally click on the wrong thing during regular play or finals.  I find it sort of nice to switch back and forth.  I like your look, nice and clean.